scalableminds/webknossos
# enable admins to edit dataset default configurations

### Mailable description of changes:
 - Admins can now edit dataset default configurations

### Steps to test:
- Click to edit a dataset, change some settings (e.g. position and zoom), save
- Open the dataset in view mode
- Position and Zoom should be preset
- Initial dataset import should still work (w/o configuration options)
- new api routes GET and POST `/api/dataSetConfigurations/default/:dataSetName`

### Issues:
- fixes #2457

------
